pkgs/tools/misc/loadlibrary/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchurl, cabextract, glibc_multi, fetchFromGitHub, readline, stdenv_32bit }:
+
+stdenv_32bit.mkDerivation {
+  name = "loadlibrary";
```

Comment:
version is missing, use something like this:

```
  name = "loadlibrary-${version}";
  version = "20170525-stdenv.lib.strings.substring 0 7 rev}";
  rev = "01hb7wzfh1s5b8cvmrmr1gqknpq5zpzj9prq3wrpsgg129jpsjkb";

  src = fetchFromGitHub {
     inherit rev;
     owner = "taviso";
     repo = "loadlibrary";
     sha256 = "01hb7wzfh1s5b8cvmrmr1gqknpq5zpzj9prq3wrpsgg129jpsjkb";
  };
```

The same we use for go packages.